lilypad/faqs.md

```diff
@@ -57,7 +57,18 @@ Phases 2 and onward will provide rewards for Lilypad modules created as well as
 
 ### How do I check my Lilybit\_ rewards?
 
-You can check your rewards by pasting your nodes wallet address into the following interfaces:
+You can check your rewards by pasting your nodes wallet address into the following inteerfaces:
+
+### How does Lilypad use blockchain, and why do I need both ETH and Lilypad tokens to run a job?
+
+On the Lilypad network, The blockchain is used for
+* payment rails
+* storing the deals transparently (on-chain guarantees about the compute)
+* storing any disputes & results
```

Comment:
```suggestion
* storing any disputes and results
```
We may want to clarify "results" here. We store metadata about a completed job on chain, but not the actual job outputs. Maybe "results metadata"? Not quite sure, and maybe I'm overthinking it.